### ISSUE TITLE
unfolds_in

### ISSUE THREAD

**Issue by zhengj2007** [BFO-ontology/bfo-trunk#77]
_From [mcour...@gmail.com](https://code.google.com/u/116795168307825520406/) on June 28, 2012 13:04:40_

I didn't find the relation unfolds_in (P unfolds_in C : the execution of P is spatially contained by C. forall pP,  forall c' partipates_in p, c' located_in C at t, for some t in P) that was in ro_proposed.
I use it in AERO to say things such as "encephalitis unfolds_in brain", i.e. linked a process with the place in which it occurs.
Did I miss something, or was it left out on purpose?

_Original issue: http://code.google.com/p/bfo/issues/detail?id=78_

**Comment by zhengj2007** [comment]
_From [cmung...@gmail.com](https://code.google.com/u/115850439786749293234/) on June 28, 2012 12:52:25_

relabeled as "occurs_in", did have BFO:0000066

**Comment by zhengj2007** [comment]
_From [mcour...@gmail.com](https://code.google.com/u/116795168307825520406/) on June 28, 2012 13:15:14_

BFO:0000066 is in the current bfo.owl "occupies", defined as "a occupies r. This is a primitive relation between an occurrent and a temporal or spatiotemporal region which it exactly occupies. (axiom label in BFO2 Reference: [082-001])"
While domain is occurrent, range is temporal or spatiotemporal regions, whereas the brain in my example above would be neither of those I think, but rather an independent continuant. 
I didn't find a mention of an "occurs" object property in the BFO2 reference document, but mentions here and there, such as Axiom: If a is an occurrent and b is an occurrent and a s-depends on b at t, then b occurs at t. [130-001]]

**Comment by zhengj2007** [comment]
_From [cmung...@gmail.com](https://code.google.com/u/115850439786749293234/) on June 28, 2012 13:38:56_

argh, can you use a different IRI. BFO_0000066 has been MIREOTed into the current RO as "occurs_in" and has been in use in GO for a number of years. See: http://www.ontobee.org/browser/rdf.php?o=GO&iri=http://purl.obolibrary.org/obo/BFO_0000066 The sense in which it is used in GO is definitely weaker. roughly: p occurs_in c => the boundaries of p are within the boundaries of c throughout the duration of the process.

I would rather BFO_0000066 was not re-used. I would ideally like to reserve it, but can live with an obsoletion if the original is problematic.

Thanks for catching this

**Comment by zhengj2007** [comment]
_From [mcour...@gmail.com](https://code.google.com/u/116795168307825520406/) on June 28, 2012 13:56:55_

If we decide to add occurs_in, inverse in RO is currently "has site of" with ID BFO:0000067 (which as far as I can see is not used atm in the bfo.owl file). 
I will create a separate issue of type bfo-owl for the IRI issue.

**Comment by zhengj2007** [comment]
_From [alanruttenberg@gmail.com](https://code.google.com/u/alanruttenberg@gmail.com/) on June 28, 2012 19:47:11_

Trying to track down how this happened, and will try to fix.  I concur that ..66 shouldn't be reused in this way, and is probably due to an error on my part. TBH I don't like either the relation or the label for this term. There should be separate relations for temporal and spatiotemporal. The spans relation seems to mean the temporal part of this relationship. Second "occupies" has too much meaning in the continuant world to use as a label for occurrent stuff.

Regarding unfolds_in I think that you are right that is missing but will double-check.

Thanks -a

**Comment by zhengj2007** [comment]
_From [alanruttenberg@gmail.com](https://code.google.com/u/alanruttenberg@gmail.com/) on June 28, 2012 20:30:16_

OK, indeed an error of mine. I have restored the id from OWL-schulz (which I mistook as a duplicate). It is now BFO_0000130 (you will see it when I next check in). I added BFO_0000066 back into bfo-uris.lisp as (:maybe ro). 

We should keep this issue open until we resolved whether to include 'occurs in'. For the definition I would rephrase (in terms of bfo2 - lisp names) as:  

p occurs-in c => 

exists st
forall(t) exists_at(p,t) -> exists_at(c,t) and 
    exists r1 , r2 and 
    p occupies st, st st-projects-on-s r1 at t, c located-at-r r2 at t, r1 c-ppart-of r2 paraphrasing: 

There is a spatiotemporal region st that p occupies.
At every time t that p exists, c exists and 
there are regions r1 and r2 where r1 is the spatial projection of st at t r2 is the region that c is located at at t
and r1 is a proper continuant part of r2
